docs/index.md

```diff
@@ -31,7 +31,8 @@ Spark runs on both Windows and UNIX-like systems (e.g. Linux, Mac OS). It's easy
 locally on one machine --- all you need is to have `java` installed on your system `PATH`,
 or the `JAVA_HOME` environment variable pointing to a Java installation.
 
-Spark runs on Java 8+, Python 2.7+/3.4+ and R 3.1+. For the Scala API, Spark {{site.SPARK_VERSION}}
+Spark runs on Java 8+, Python 2.7+/3.4+ and R 3.1+. R prior to version 3.4 is deprecated as of Spark 3.0.
```

Comment:
3.1+ -> 3.4+?

Reply:
Hmmmmm .. I was thinking we could change them when we actually drop the support. Technically it does support 3.1+ yet although 3.1, 3.2, and 3.3 are deprecated.

Reply:
hmm, so R prior to version 3.4 is just deprecated, not dropped in in Spark 3.0?

Reply:
Ah,  yea, I switched this to deprecate it for now. I was a bit curious about that.

Reply:
At least Python version was noted here before.